sparse_strips/vello_cpu/src/fine/mod.rs

```diff
@@ -93,53 +124,141 @@ impl<'a> Fine<'a> {
     }
 
     /// Fill at a given x and with a width using the given paint.
-    pub fn fill(&mut self, x: usize, width: usize, paint: &Paint) {
-        let scratch = self.scratch.last_mut().unwrap();
-        match paint {
-            Paint::Solid(c) => {
-                let color = c.to_u8_array();
-
-                let target =
-                    &mut scratch[x * TILE_HEIGHT_COMPONENTS..][..TILE_HEIGHT_COMPONENTS * width];
+    pub fn fill(
+        &mut self,
+        x: usize,
+        tile_x: u16,
```

Comment:
`fill`, `strip` (and thus `run_cmd`), and `pack` all need to know the tile coordinates now. Might be worth storing in `fine` instead?